src/options/settings/components/SearchInjection.js

```diff
@@ -14,10 +14,11 @@ const SearchInjection = ({ children }) => {
                 Want others?{' '}
                 <a
                     target="_new"
-                    href="https://github.com/WorldBrain/Memex/blob/master/CONTRIBUTING.md"
+                    href="https://github.com/WorldBrain/Memex/blob/ee8ea586b3b5101fd18e5f06659f56b7b97fda0b/CONTRIBUTING.md"
```

Comment:
This link is absolute, this module has been renamed. The correct link should be: https://github.com/WorldBrain/Memex/blob/master/GETTING-STARTED.md